src/react-vite/02-setting-up-your-environment/setting-up-your-environment.md

```diff
@@ -6,14 +6,6 @@
 
 @body
 
-## How to follow this course
-
-This course will walk you through building an application with React. Each page of the guide is based on building a new feature, and may have multiple “objectives” to complete. Each objective will be explained and include the requirements and any setup steps needed. Most objectives will also include unit tests to update to verify when you have implemented the solution correctly.
-
-The ✏️ icon will be used to indicate when commands need to be run or when files need to be created or updated.
-
-If you have any issues or suggestions as you move through this training, we’d love for you to submit a <a href="https://github.com/bitovi/academy/issues/new">GitHub issue</a> for it! 💖
-
```

Comment:
Please keep this in the content.

Reply:
It was moved to the beginning of the training.